tests/common.py

```diff
@@ -84,15 +85,8 @@ def get_git_revision_short_hash() -> str:
     db.MsSQL: TEST_MSSQL_CONN_STRING,
 }
 
-_database_instances = {}
-
-
-def get_conn(cls: type, shared: bool = True) -> Database:
```

Comment:
We should be reusing this connection vs. creating a new one each time. This change should be reverted.

Reply:
Can you point me to the failed test logs where you verified this is causing the issue?

Reply:
Actually, I think I found it when locally testing. I'll update this thread in a bit.